### PR TITLE
frontend: Better define nodes for baremetal

### PR DESCRIPTION
**Pull request by ggreer**
Put node count inputs for masters/workers on master/worker pages, respectively. This is a much better UI.

### CODE REVIEW

File: installer/frontend/components/bm-nodeforms.jsx [hunk 1]
```diff
@@ -67,7 +75,7 @@ const BulkUpload = connect(
         });
       },
       dirtyAllFields: (label, nodes) => {
-        nodes.map((_,i) => {
+        nodes.map((v, i) => {
```
Comment (kans) [inline installer/frontend/components/bm-nodeforms.jsx]: I think eslint supports "unused" for the variable name.

File: installer/frontend/components/bm-nodeforms.jsx [hunk 2]
```diff
@@ -31,126 +39,102 @@ const countBy = (collection, f) => {
...
-      },
+// TODO (ggreer) make a real modal with real modal classes
+class BulkUpload extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      name: null,
+      macCol: 0,
+      nameCol: 1,
+      columns: [],
```
Comment (kans) [inline installer/frontend/components/bm-nodeforms.jsx]: unused

**Comment by kans** [comment]
What happens if you give it a pdf?

**Comment by ggreer** [comment]
It dutifully parses it as best it can:
screen shot 2017-05-10 at 14 04 27

**Pull request merged**